specs/file_hierarchy_for_the_verification_of_os_artifacts.md

```diff
@@ -0,0 +1,639 @@
+---
+title: File Hierarchy for the Verification of OS Artifacts (VOA)
+category: Concepts
+layout: default
+version: 1
+SPDX-License-Identifier: CC-BY-4.0
+---
+
+# File Hierarchy for the Verification of OS Artifacts (VOA)
+
+## Motivation
+
+Cryptographic validation of artifacts with the help of digital signatures is a use-case of most Linux distributions.
+Different cryptographic technologies exist and can be used for this purpose.
+Currently, OpenPGP and X.509 are widely adopted.
+
+As of this writing, no technology-agnostic, standardized location for the distribution of cryptograpic material that serves as verifier for digital signatures exists.
+This leaves consumers to either do guesswork, or rely on proprietary, stateful or technology-specific keystore formats and per-application locations.
+
+The _File Hierarchy for the Verification of OS Artifacts (VOA)_ defines a generic approach for storage and retrieval of _signature verifiers_, supporting a wide range of cryptographic technologies.
+
+VOA aims to cover all verification needs around OS artifacts.
+
+Unlike e.g. Mozilla's Network Security Services ([NSS]), which concerns itself with secure network communication, including a curated set of X.509 CA certificates (e.g. often provided on Linux distributions via `/etc/pki` or `/etc/ca-certificates`), VOA focusses on the verification of artifacts with support for a diverse set of cryptographic technologies.
```

Comment:
not that it matters, but in systemd docs we generally insist on suffixing paths that refer to dirs with a `/`, simply to visually communicate that we are talking of a dir here.

(just an idea, no need to follow systemd's rules here, but we have them for a reason, a reason I think makes sense)

Reply:
Thanks for finding the remaining outliers!
(all the other directory names already behaved as you described :sweat_smile: )